apps/frontend-manage/src/pages/sessions/[id]/evaluation.tsx

```diff
@@ -207,9 +207,9 @@ function Evaluation() {
 
       <div
         className={twMerge(
-          'flex justify-between flex-col mt-11 [height:_calc(100%-6.25rem)] mb-14',
+          'flex justify-between flex-col mt-11 [height:_calc(100%-6.25rem)] pb-14',
           (showFeedbacks || showConfusion || showLeaderboard) &&
-            '[height:_calc(100%-7.25rem)] mb-18'
+            '[height:_calc(100%-7.25rem)] pb-18'
```

Comment:
Removing the margin and adding a padding seems to lead to issues in some cases with a white bar inside the evaluation area. We should investigate, if the margin did indeed work better or if there is some other kind of issue as well.

<img width="1517" alt="Screenshot 2023-02-19 at 12 39 08" src="https://user-images.githubusercontent.com/80708107/219946051-4740c10f-97d7-4629-9453-b16e35de4e73.png">
<img width="1391" alt="Screenshot 2023-02-19 at 12 45 28" src="https://user-images.githubusercontent.com/80708107/219946054-7e69eed2-23b3-43ff-bce6-728668273bd2.png">

Reply:
Issue does not occur, when the questions are larger and displayed with a collapsible